src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/AuthorizationServer/ApiAccess/form_theme.html.twig

```diff
@@ -0,0 +1,19 @@
+{% use '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig' %}
+
+{% block api_access_widget %}
+  <div class="card-body">
+    <div class="form-wrapper">
+      {{ block('form_widget') }}
+
+      {% if form.vars.form.vars['empty_scopes'] == true %}
+        <script>
+          const col = document.getElementById('api_access_scopes');
+          col.style.height = col.parentElement.offsetHeight + "px";
+          col.innerHTML = "<div id='scope_disclaimer' class='d-flex align-items-center'><p class='align-middle mb-0'>No scopes available</p></div>";
+          const disclaimer = document.getElementById('scope_disclaimer');
+          disclaimer.style.height = col.parentElement.offsetHeight + "px";
+        </script>
+      {% endif %}
```

Comment:
Why do you need all this complex js? A simple HTML layout should be enough, and the text should be translated
```suggestion
        <p class="text-start">{{ 'No scopes available'|trans({}, 'Admin.Advparameters.Feature') }}</p>
      {% endif %}
```

Reply:
Well you need something. A simple html layout is not working. The added text is not in the right place at all. 

In order for the text to be on the same height thant his label, you need to insert it in the correct div. 

Then the div you insert it into has a half height of it's parent so the text is not verticaly align.